django_tasks/backends/database/backend.py

```diff
@@ -28,13 +30,14 @@ class DatabaseBackend(BaseTaskBackend):
     supports_async_task = True
     supports_get_result = True
     supports_defer = True
+    supports_signed_task = True
 
     def _task_to_db_task(
         self, task: Task[P, T], args: P.args, kwargs: P.kwargs
     ) -> "DBTaskResult":
         from .models import DBTaskResult
 
-        return DBTaskResult(
+        db_task_result: DBTaskResult = DBTaskResult(
```

Comment:
Nit: The explicit type shouldn't be necessary here
```suggestion
        db_task_result = DBTaskResult(
```

Reply:
mypy yelled about it, it really should be able to infer that.